t/001compile.t

```diff
@@ -20,16 +20,20 @@ use Support::Files;
 use Test::More;
 
 BEGIN {
-  if ($ENV{CI}) {
-    plan skip_all => 'Not running compile tests in CI.';
-    exit;
-  }
   plan tests => @Support::Files::testitems + @Support::Files::test_files;
 
   use_ok('Bugzilla::Constants');
   use_ok('Bugzilla::Install::Requirements');
   use_ok('Bugzilla');
+
+  if (-f 'data/db/model_test') {
+    unlink 'data/db/model_test' || die $!;
```

Comment:
This needs to be either `unlink('data/db/model_test') || die $!` or `unlink 'data/db/model_test' or die $!`